assets/src/components/routeLadder.tsx

```diff
@@ -48,39 +42,6 @@ interface Props {
   onAddDetour?: (route: Route) => void
 }
 
-export const Header = ({
-  routeName,
-  onClose,
-  hasAlert,
-}: {
-  routeName: string
-  onClose: () => void
-  hasAlert: boolean
-}) => {
-  return (
-    <div className="c-route-ladder__header">
-      {hasAlert && (
-        <Tippy
-          content="Active detour"
-          trigger="click"
-          onShow={() => tagManagerEvent("alert_tooltip_clicked")}
-        >
-          <AlertIcon
-            className="c-route-ladder__alert-icon"
-            aria-label="Route Alert"
-          />
-        </Tippy>
-      )}
-      <div className="c-route-ladder__close-button-container">
-        <OldCloseButton closeButtonType="l_darker" onClick={onClose} />
-      </div>
-
-      <div className="c-route-ladder__route-name">{routeName}</div>
-    </div>
-  )
-}
-
-// TODO: delete old header after roll-out
 export const NewHeader = ({
```

Comment:
What do you think about renaming `NewHeader` to `Header` in this PR? I think it should be renamed with this work, but understand if you'd want it in a new PR

Reply:
Oooh additionally, beyond just changing the component name, the "new" label on the classnames will need changing. In hindsight, this should have been done in my work! I should have used the term "oldHeader" instead of going with "newHeader"

Reply:
I created a [follow-up subtask](https://app.asana.com/0/1205385723132845/1207827639549180/f) for this in order to keep the diff for this smaller and easier to review. I could lump them together if you'd prefer.